File: patterns/pattern3.c
/*
5
4 4
3 3 3
2 2 2 2
1 1 1 1 1
*/
#include<stdio.h>
int main()
{
	int rows,i,j;
	printf("enter the no.of rows:\n");
	scanf("%d",&rows);
	for(i=rows;i>0;i--,printf("\n"))
	{
		for(j=rows;j>=i;j--)
		{
			printf("%d ",i);
		}
		
	}
}

File: PYTHON1/pattern3.py
'''
5
4 4
3 3 3
2 2 2 2
1 1 1 1 1
'''
rows=int(input("enter the no.of rows:"))
num=5
for i in range(rows,0,-1):
    for j in range(rows,i-1,-1):
        print(num ,end=" ")
    print("\n")
    num=num - 1
